src/views/projects/Header.svelte

```diff
@@ -116,4 +119,9 @@
       {baseUrl.hostname}
     </SquareButton>
   </Link>
+  <SquareButton>
```

Comment:
I would add here that `hoverable={false}` so we match the style of the `contributors` label